pkg/trace/writer/stats.go

```diff
@@ -247,6 +247,8 @@ func (w *StatsWriter) resolveContainerTags(p *pb.ClientStatsPayload) {
 	case len(ctags) == 0:
 		p.Tags = nil
 	default:
+		// TODO: Once we can deprecate the `enable_cid_stats` feature flag, let's make sure to not duplicate
```

Comment:
Is this on a roadmap somewhere or is there a like one-pager or something for deprecating this feature flag?

Reply:
I would say this could be the start of the roadmap 😅 

This flag and the associated implementation was done a while back. We need to talk offline about re-prioritizing the push to move this flag to become a default behavior.